xchange-quadrigacx/src/main/java/org/knowm/xchange/quadrigacx/QuadrigaCxUtils.java

```diff
@@ -28,9 +30,11 @@ private QuadrigaCxUtils() {
    * @return
    */
   public static Date parseDate(String dateString) {
-
     try {
-      return DATE_FORMAT.parse(dateString);
+        // SimpleDateFormat is not thread safe, there fore create a new instance every time
```

Comment:
sorry, this change is not part of this pull request any more, was my fault...

quadriga is now on the other pull request
https://github.com/timmolter/XChange/pull/1333

but to answer your question: sore, one can use thread local here, but i'm not sure whether its worth it...
is the creation of SimpleDateFormat that expensive ?